contrib/cirrus/test.sh

```diff
@@ -51,11 +69,13 @@ else
             showrun make test-conformance
             ;;
         integration)
-            # FIXME: drop the `rm` below once containers.conf has been fixed.
-            # It complains about failing to "to decode the keys ["secret"
-            # "secret.opts"]" which is in process of getting fixed but will
-            # take a while until it hits all distributions.
-            showrun rm /usr/share/containers/containers.conf
+            if [[ "$PRIV_NAME" != "rootless" ]]; then
+               # FIXME: drop the `rm` below once containers.conf has been fixed.
+               # It complains about failing to "to decode the keys ["secret"
+               # "secret.opts"]" which is in process of getting fixed but will
+               # take a while until it hits all distributions.
+               showrun rm /usr/share/containers/containers.conf
+            fi
```

Comment:
Might be worth trying without this block, I don't know when it was added or what the issue was, but it may already be fixed.  Ya, this is causing the tests to fail when running as a regular user: permission denied. If this removal is still necessary, perhaps move it into setup.sh where we still execute as root?